### ISSUE TITLE
Two equivalent stubs using different syntax

### ISSUE THREAD

**Issue by bquorning** [backus/rubocop-rspec#392]
I just came across a spec that had (the equivalent of) the following lines:
```rb
allow(:foo).to receive(:bar) { "baz" }
allow(:foo).to receive(:bar).and_return("baz")
```

It would be nice if I had a cop to detect this kind of duplication. Note that there’s a subtle difference between the two lines, but in this case it doesn’t matter.

**Comment by backus** [comment]
Yes I was thinking this recently. It would be nice to also enforce one style (except in cases where the value inside of the block in the first style is dynamic).

**Comment by Darhazer** [comment]
I can implement such. The one of the supported option is obviously `block`; how should be called the other one?

**Comment by Drenmi** [comment]
Can this be closed as of the merging of #422? 🙂
